src/main/java/com/conversantmedia/util/concurrent/AbstractWaitingCondition.java

```diff
@@ -212,4 +250,17 @@ public void signal() {
             }
         }
     }
+
+    public long sumToAvoidOptimization() {
+        return // LhsPadding r1-r15
+                 r11+r12+r13+r14+r15+r16+r17+r18         +r21+r22+r23+r24+r25+r26+r27+r28         +r31+r32+r33+r34+r35+r36+r37+r38         +r41+r42+r43+r44+r45+r46+r47+r48
```

Comment:
consider formatting this as 2 or even one columns?   I think it would be more legible on GitHub